src/databases/MFEM/CMakeLists.txt

```diff
@@ -3,6 +3,8 @@ PROJECT(MFEM_database)
 
 INCLUDE(${VISIT_SOURCE_DIR}/CMake/PluginMacros.cmake)
 
+ADD_DATABASE_CODE_GEN_TARGETS(MFEM)
```

Comment:
No vtk-h related dependencies here?

Reply:
again - this is just the result of our current xml2cmake. 

Pretty sure, plots and operators need to know about vtk-h and vtk-m, but not databases.

Reply:
Only those plots and operators actually utilizing vtk-h need the includes, but they were added to the code gen to make it easier for development of future plugins (?). IMHO, I think they should be added to the xml files of those plugins requiring them instead of applied to all plugins via code gen of CMakeLists.txt

Reply:
I agree, we can add a ticket to follow up.

Reply:
> Only those plots and operators actually utilizing vtk-h need the includes, but they were added to the code gen to make it easier for development of future plugins (?). IMHO, I think they should be added to the xml files of those plugins requiring them instead of applied to all plugins via code gen of CMakeLists.txt

Perhaps the right answer here is to recognize when code-gen tools are being used to *jump-start* development of a new plugin vs. re-gening existing code. I think it makes sense for the *jump-start* mode to possibly include (or via comments suggest including) commonly needed header files, etc. But, that service is really to help newbie/novice developers over common cases and is not something we want to impact *mature* code. So, maybe we should have an additional CL switch to the code-gen tools that disables this from within these targets but which is otherwise on by default. That would address the reasons this boilerplate code is in the code-gen outputs to begin with but would prevent it from polluting re-gens of mature code. Maybe a new issue ticket?

Reply:
Note I added: #4179

Reply:
My specific observation is: there isn't a compelling case to have them at all, given they are core-ish dependencies -- they should be included by indirection.